examples/src/uds/server.rs

```diff
@@ -26,8 +29,13 @@ impl Greeter for MyGreeter {
     ) -> Result<Response<HelloReply>, Status> {
         #[cfg(unix)]
         {
-            let conn_info = request.extensions().get::<unix::UdsConnectInfo>().unwrap();
+            let conn_info = request.extensions().get::<UdsConnectInfo>().unwrap();
             println!("Got a request {:?} with info {:?}", request, conn_info);
+
+            // Client-side unix sockets are unnamed.
+            assert!(conn_info.peer_addr.as_ref().unwrap().is_unnamed());
+            // This should contain process credentials for the client socket.
+            assert!(conn_info.peer_cred.as_ref().is_some());
```

Comment:
Probably makes sense to have these as doc items and not have asserts in an example, what do you think?

Reply:
Sounds good to me 👍🏽 there's already sufficient documentation on these fields in the `UdsConnectInfo` struct, so I'll just go ahead and remove these assertions - the info will be printed by the statement just above anyways.

Reply:
Yup that works for me then we can merge! Thank you for the patience on this!

Reply:
@LucioFranco For sure! Thanks for taking the time to review.

Fixup commits have been squashed - hopefully we're good to merge now 👀